cluster/manifests/postgres-operator/credentials.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "zalando.org/v1"
+kind: PlatformCredentialsSet
+metadata:
+  name: postgresql-operator
+spec:
+  application: postgresql-operator
+  tokens:
+    read-only:
+      privileges:
+    cluster-registry-rw:
```

Comment:
I'm wondering why the operator, which is per cluster, needs access to the cluster-registry?